apps/api-server/src/models/Project.js

```diff
@@ -267,7 +266,10 @@ module.exports = function (db, sequelize, DataTypes) {
     let self = this;
     let voteIsActive = self.config.votes.isActive;
     if ( ( voteIsActive == null || typeof voteIsActive == 'undefined' ) && self.config.votes.isActiveFrom && self.config.votes.isActiveTo ) {
-      voteIsActive = moment().isAfter(self.config.votes.isActiveFrom) && moment().isBefore(self.config.votes.isActiveTo)
+      let isActiveFromTimestamp = new Date( self.config.votes.isActiveFrom ).getTime();
+      let isActiveToTimestamp = new Date( self.config.votes.isActiveTo ).getTime();
+      let nowTimestamp = new Date().getTime;
```

Comment:
In your comparison nowTimestamp resolves to the function, not the value it keeps.

Reply:
En daarom is het zo fijn als er iemand meekijkt :)